TripKitDomain/src/main/java/skedgo/tripkit/geocoding/Geocodable.kt

```diff
@@ -1,7 +0,0 @@
-package skedgo.tripkit.geocoding
-
-import rx.Observable
-
-interface Geocodable {
-  fun getCoordinates(address: String): Observable<Pair<Double, Double>>
-}
```

Comment:
I changed the name from `GeoCodable` into `Geocodable` to match with the definition from wiki https://en.wikipedia.org/wiki/Geocoding. So can we reuse this interface?